main/index.js

```diff
@@ -33,6 +37,22 @@ const initializePlugins = async () => {
   }
 };
 
+const checkForUpdates = () => {
+  if (is.development) {
+    return false;
+  }
+
+  // For auto-update debugging in Console.app
+  autoUpdater.logger = log;
+  autoUpdater.logger.transports.file.level = 'info';
+
+  setInterval(() => {
+    autoUpdater.checkForUpdates();
```

Comment:
I think this should use `checkForUpdatesAndNotify`? 

>Asks the server whether there is an update, download and notify if update available.


https://www.electron.build/auto-update

Reply:
#481, #451